src/routes/transactions/helpers/transaction-verifier.helper.spec.ts

```diff
@@ -305,40 +304,9 @@ describe('TransactionVerifierHelper', () => {
             safe,
           });
 
-        await expect(
-          target.verifyApiTransaction({ chainId, safe, transaction }),
-        ).resolves.not.toThrow();
-      });
-
-      it.each([
-        SignatureType.ApprovedHash as const,
-        SignatureType.ContractSignature as const,
-      ])('should validate a %s signature', async (signatureType) => {
```

Comment:
As these signatures are now in the builder, they are covered by the `it.each(Object.values(SignatureType))('...` changes.